components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -890,7 +889,11 @@ SUBROUTINE p3_main_part2(kts, kte, kbot, ktop, kdir, do_predict_nc, do_prescribe
          is_hydromet_present = .true.
       endif
 
-      call impose_max_total_ni(ni(k),max_total_ni,inv_rho(k))
```

Comment:
@JS-WRF-SBM - your code doesn't have this change. I think max_total_ni must be an in-cloud quantity... or at least it can't simultaneously be a limit for in-cloud and cell-ave!

Reply:
Thanks!